backend/Spring Boot/springboot_starterproject/README.md

```diff
@@ -0,0 +1,29 @@
+# Spring Boot Starter Project
+
+## 📌 Introduction
+This Project contains basic template for Spring Boot.<br>
+
+
+## 📌 What You will build using this project
+You will build a simple web application with Spring Boot.<br>
+There will be a health API to check project is working correctly.<br>
+
+## 📌 What You will need
+IntelliJ IDEA<br>
+Java installed with version 8 or higher<br>
+Maven<br>
+
+## 📌 Ways to create Spring Boot Starter Project
+Using Spring.io initializer<br>
+Using IntelliJ or any similar IDE and Maven simple project<br>
+Using Spring Tool Suite <br>
+Using CLI<br>
+
+
+

```

Comment:
Extra lines. I guess you have added this everywhere, so I'm not commenting on it everywhere now. Please do this spacing change. Also, keep only a single blank line when starting new headers.